lib/importer/index.js

```diff
@@ -206,15 +207,26 @@ exports.Initialise = (config, router, prototypeKit) => {
 
 };
 
+//--------------------------------------------------------------------
+// Where the user has not configured an upload path in their prototype's 
+// config.json, we will instead create one.  
+//--------------------------------------------------------------------
+const makeTempStorage = () => {
+  const path = require('node:path');
+  const os = require('node:os');
+  return fs.mkdtempSync(path.join(os.tmpdir(), 'reg-dyn-importer')); 
+}
+
 const getUploader = (config) => {
   const storage = multer.diskStorage({
     destination: (req, file, cb) => {
-      var fs = require("fs");
-      if (!fs.existsSync(config.uploadPath)) {
-        fs.mkdirSync(config.uploadPath, { recursive: true });
+      uploadPath = config.uploadPath ??= makeTempStorage();
```

Comment:
You can just use `??` here?

Reply:
I could, but I was cheating here to set config.uploadPath at the same time as it is also used elsewhere :)